modules/apps/layout/layout-reports-web/package.json

```diff
@@ -15,6 +18,11 @@
 		"react": "16.12.0",
 		"recharts": "1.8.5"
 	},
+	"jest": {
+		"setupFiles": [
+			"<rootDir>/jest-setup.config.js"
+		]
+	},
```

Comment:
(Outside the scope of this PR, but this is another thing that is more or less repeated verbatim in about 10 places in `liferay-portal` now. Seems like the kind of thing we might be able to abstract away... Just throwing that out there in case anybody reading this feels like writing up a ticket for it — I'm not totally convinced it's a good idea, so I'm not going to right now).